### ISSUE TITLE
How can I use Greasemonkey functions to copy some modified from the selected text?

### ISSUE THREAD

**Issue by pendave** [SanderRonde/CustomRightClickMenu#45]
![image](https://user-images.githubusercontent.com/1331881/76722352-2f942680-677e-11ea-9b27-71135732ff33.png)

I try to do this way but failed.

What I wanna to is select text such as 

> we have recently completed depicting the #COVID2019 #Coronavirus at the centre of the

from the webpage, and CopyMine will le me get a clipboard content of 

> we have recently completed depicting the COVID2019 Coronavirus at the centre of the

just remove some characters I don't want!.

**Comment by SanderRonde** [comment]
You mean that what you want is the second thing you posted and the original is the first thing right? Or is the second thing what is being copied while the first thing is what's the original?

**Comment by pendave** [comment]
Yes, the first thing is a simple test using Greasemonkey
`GM.setClipboard('AAAA');`

The second step will be 
`GM.setClipboard('AAAA'.replace(/#/g, ''));`

**Comment by SanderRonde** [comment]
So which part is failing? Is it not copying anything at all or is the copied content incorrect?

**Comment by pendave** [comment]
It is not copying anything at all.

**Comment by SanderRonde** [comment]
So I looked into the code and it turns out I didn't implement that function because I thought it wasn't possible at the time. I did implement it now but I doubt this change will make it to the live version quickly due to the whole chrome web store mess and them still not reinstating the extension.

In the meantime I recommend you use the following code:
```js
const element = document.createElement('span');
element.innerText = 'my text';
document.body.appendChild(element);

const snipRange = document.createRange();
snipRange.selectNode(element);
const selection = window.getSelection();
selection.removeAllRanges();
selection.addRange(snipRange);
try {
	document.execCommand('copy');
} catch(e) {
	console.log(e);
}

selection.removeAllRanges();
element.remove();
```

Where `'my text'` is the data you want to copy to the clipboard

**Comment by SanderRonde** [comment]
Fixed in 70f629fa15fb4053ba943914be93e8dcfa9d7819
